gems/actionpack/6.0/actioncontroller.rbs

```diff
@@ -1,6 +1,5 @@
 module ActionController
   interface _API_and_Base
-    def session: () -> untyped
```

Comment:
I cannot determine if this change will not break compatibility.
@pocke Do you know what it is?

Reply:
I think the signature of `ActionController::Base#session` and `ActionController::API#session` have not changed with this change.

Before:
```
tkomiya@tarf> bundle exec rbs method ActionController::Base session
::ActionController::Base#session
  defined_in: ::ActionController::_API_and_Base
  implementation: ::ActionController::Base
  accessibility: public
  types:
      () -> untyped   at /Users/tkomiya/work/rbs/gem_rbs_collection/gems/actionpack/6.0/_test/.gem_rbs_collection/actionpack/6.0/actioncontroller.rbs:3:17...3:30
```
```
tkomiya@tarf> bundle exec rbs method ActionController::API session                                                                                      ~/work/rbs/gem_rbs_collection/gems/actionpack/6.0/_test [main]
::ActionController::API#session
  defined_in: ::ActionController::_API_and_Base
  implementation: ::ActionController::API
  accessibility: public
  types:
      () -> untyped   at /Users/tkomiya/work/rbs/gem_rbs_collection/gems/actionpack/6.0/_test/.gem_rbs_collection/actionpack/6.0/actioncontroller.rbs:3:17...3:30
```

After:
```
tkomiya@tarf> bundle exec rbs method ActionController::Base session
::ActionController::Base#session
  defined_in: ::ActionController::Metal
  implementation: ::ActionController::Metal
  accessibility: public
  types:
      () -> untyped   at /Users/tkomiya/work/rbs/gem_rbs_collection/gems/actionpack/6.0/_test/.gem_rbs_collection/actionpack/6.0/patch.rbs:5:17...5:27
```
```
tkomiya@tarf> bundle exec rbs method ActionController::API session
::ActionController::API#session
  defined_in: ::ActionController::Metal
  implementation: ::ActionController::Metal
  accessibility: public
  types:
      () -> untyped   at /Users/tkomiya/work/rbs/gem_rbs_collection/gems/actionpack/6.0/_test/.gem_rbs_collection/actionpack/6.0/patch.rbs:5:17...5:27
```

Reply:
It probably does not break compatibility.
As @tk0miya said, `ActionController::Base` and `API` have the `session` method even if this patch is applied.

If a user uses `ActionController::_API_and_Base` interface in their app directly, it can be a breaking change. But I do not intend such usage. Therefore we can ignore the impact.